cmd/clusterctl/client/tree/annotations.go

```diff
@@ -58,7 +58,7 @@ const (
 
 // GetMetaName returns the object meta name that should be used for the object in the presentation layer, if defined.
 func GetMetaName(obj client.Object) string {
```

Comment:
I don't really mind, but this change is 100% unrelated, right?

Reply:
A prior iteration had new convenience funcs in this file, this is leftover, I'll remove!